src/dodal/devices/areadetector/plugins/MJPG.py

```diff
@@ -39,9 +40,13 @@ def _save_image(self, image: Image.Image):
         and filename signals. The full resultant path is put on the last_saved_path signal
         """
         filename_str = self.filename.get()
-        directory_str = self.directory.get()
+        directory_str = Path(self.directory.get())  # type: ignore
```

Comment:
NIT: Maybe misleading variable name since it's type Path not string. Also, nothing complains for me if I remove the type ignore